tdrs-frontend/src/actions/reports.js

```diff
@@ -0,0 +1,5 @@
+export const SET_YEAR = 'SET_YEAR'
+
+export const setYear = (year) => (dispatch) => {
+  dispatch({ type: SET_YEAR, payload: { year } })
+}
```

Comment:
Alex, I think you were right that the naming of this action was confusing. I changed the file name to `reports` and that will have multiple different actions in it. If we continue creating a new action for every new aspect, I think they are going to get a bit unmanageable so I'm going to start combining them as much as possible. I'm happy to discuss this further! 

This is to set the year in the Redux store under the `Reports` module.

Reply:
@hilvitzs Great!